client/src/components/graphs/OrganizationalChart.tsx

```diff
@@ -189,260 +151,608 @@ const OrganizationalChart = ({
     pageDispatchers
   })
 
-  const canvas = d3.select(canvasRef.current)
-  const link = d3.select(linkRef.current)
-  const node = d3.select(nodeRef.current)
-  const attachmentsEnabled = !Settings.fields.attachment.featureDisabled
-
-  useEffect(() => {
-    data &&
-      setRoot(
-        d3.hierarchy(data.organization, d =>
-          expanded.includes(d.uuid)
-            ? data.organization.descendantOrgs.filter(
-              org => org.parentOrg?.uuid === d.uuid
-            )
-            : null
-        )
-      )
-  }, [data, expanded])
+  if (done) {
+    return result
+  }
+
+  if (!data || !data.organization) {
+    return <p>Loading...</p>
+  }
+
+  return (
+    <ReactFlowProvider>
+      <OrganizationFlowChart data={data} exportTitle={exportTitle} />
+    </ReactFlowProvider>
+  )
+}
 
-  useEffect(() => {
-    if (!data || !root) {
+interface OrganizationFlowChartProps {
+  data: { organization: { uuid: string } }
```

Comment:
This type is incomplete; you also use `ascendantOrgs` and `descendantOrgs` for objects of this type.